### PR TITLE
Make clipboard always sticky

### PR DESCRIPTION
**Pull request by purwandi**
Before

Screen Shot 2020-07-12 at 11 06 17

After

Screen Shot 2020-07-12 at 11 06 52

### CODE REVIEW

**Comment by lunchboxav** [comment]
This works! Thank you very much for your contribution @purwandi :D

File: site/themes/getenvoy/static/css/main.css [hunk 1]
```diff
@@ -2751,22 +2749,22 @@ article.repo > ul > li {
 		text-shadow: 0 0 1px rgba(0,0,0,.25);
 	}
 
-	code button{
+	pre button{
 		cursor: pointer;
 		position: absolute;
-                right: 8px;
-		top: 8px;
+    right: 12px;
```
Comment (lunchboxav) [inline site/themes/getenvoy/static/css/main.css:2755]: super nit, can we adjust the tabbing here, so the indentation format stays as before?

File: site/themes/getenvoy/layouts/partials/quickstart.html [hunk 2]
```diff
@@ -2,9 +2,7 @@
   <div id="quickstart">
     <div id="quickstart-cli">
       <h2>Fetch, run and debug Envoy using the CLI</h2>
-      <pre>
-        <code class="language-sh">$ curl -L https://getenvoy.io/cli | bash -s -- -b /usr/local/bin <br />$ getenvoy run standard:{{.Site.Params.about.envoyVersion}} -- --version</code>
-      </pre>
+      <pre><code class="language-sh">$ curl -L https://getenvoy.io/cli | bash -s -- -b /usr/local/bin <br />$ getenvoy run standard:{{.Site.Params.about.envoyVersion}} -- --version</code></pre>
```
Comment (lunchboxav) [inline site/themes/getenvoy/layouts/partials/quickstart.html:5]: can we keep the indentation?
```
<pre>
    <code>...</code>
</pre>
```

Comment (purwandi) [reply site/themes/getenvoy/layouts/partials/quickstart.html:5]: If you add enter, you will have `whitespace` problem, 

Screen Shot 2020-07-13 at 07 35 43

You can compare to https://www.getenvoy.io/install/ on `Installation` section btw

Screen Shot 2020-07-13 at 07 39 13

File: site/themes/getenvoy/static/css/main.css (hunk 1 above)
Comment (purwandi) [reply site/themes/getenvoy/static/css/main.css:2755]: arghh editorconfig problem :-P

File: site/themes/getenvoy/layouts/partials/quickstart.html (hunk 2 above)
Comment (lunchboxav) [reply site/themes/getenvoy/layouts/partials/quickstart.html:5]: Ah yes. actually, this space is part of the design decision, so please keep it as intended :)

Comment (purwandi) [reply site/themes/getenvoy/layouts/partials/quickstart.html]: done, with additional css

**Comment by lunchboxav** [comment]
Thank you very much for the kind contribution @purwandi :)

**Pull request merged**